packages/components/src/Text/Text.test.tsx

```diff
@@ -148,3 +148,17 @@ it("renders a center-aligned text", () => {
     </div>
   `);
 });
+
+it("renders underlined text", () => {
```

Comment:
I wondered whether it might be worth including a test that combines a colour with the underline style? For example, 

```html
<Text variation="error" underline="dashed color-green" size="large">
  Some horrible error message.
</Text>
```

That said, you have the test for `<Typography>`. Given how it all works, the test is probably quite unnecessary.

Reply:
Yeah I figured since it was just passing the props in, there isn't really a need to test it in both places. To Atlantis reviewers: if you feel this is needed, I can add it. Otherwise, feels like a lower value test to me but happy to be wrong.